src/index.html

```diff
@@ -7,6 +7,321 @@
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a href="#home" class="header_logo">
+        <img
+          src="/images/logo.png"
+          alt="MOYO Logo"
+          class="header_logo-img"
+        >
+      </a>
+      <nav class="header__nav nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a href="#part1" class="nav__link is-active">APPLE</a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#part2" class="nav__link">SAMSUNG</a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#part3" class="nav__link">СМАРТФОНЫ И ТЕЛЕФОНЫ</a>
+          </li>
+
+          <li class="nav__item">
+            <a
+              href="#part4"
+              class="nav__link"
+              data-qa="nav-hover"
+            >
+              НОУТБУКИ И КОМПЬЮТЕРЫ
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#part5" class="nav__link">ГАДЖЕТЫ</a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#part6" class="nav__link">ПЛАНШЕТЫ</a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#part7" class="nav__link">ФОТО</a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#part8" class="nav__link">ВИДЕО</a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+
+    <main class="main">
+      <div class="main__container">
+        <div class="card" data-qa="card">
+          <img
+            class="card__image"
+            src="/images/imac.jpeg"
+            alt="APPLE A1419 iMac 27"
+          >
+          <h2 class="card__name">
+            Моноблок APPLE A1419 iMac 27" Retina 5K (MNED2UA/A)
+          </h2>
+          <span class="card__code">
+            Код товара: 195434
+          </span>
+          <div class="card__rating">
+            <div class="stars card__stars--4">
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+            </div>
+            <span class="card__opinion">Отзывов: 5</span>
+          </div>
+          <div class="card__price">
+            <span>Цена:</span>
+            <div class="card__price-cash">69 999 грн</div>
+          </div>
+          <a href="#" class="card__buy" data-qa="card-hover">
+            Купить
+          </a>
+        </div>
+      </div>
+
+      <div class="main__container">
+        <div class="card">
+          <img
+            class="card__image"
+            src="/images/imac.jpeg"
+            alt="APPLE A1419 iMac 27"
+          >
+          <h2 class="card__name">
+            Моноблок APPLE A1419 iMac 27" Retina 5K (MNED2UA/A)
+          </h2>
+          <span class="card__code">
+            Код товара: 195434
+          </span>
+          <div class="card__rating">
```

Comment:
```suggestion
          </h2>
          
          <span class="card__code">
            Код товара: 195434
          </span>
          
          <div class="card__rating">
```
Лучше раделать код, так он лучше читается